README.md

```diff
@@ -107,6 +107,7 @@ Each rule has emojis denoting:
 | [no-arguments-for-html-elements](./docs/rule/no-arguments-for-html-elements.md)                           | ✅  |     |     |     |
 | [no-aria-hidden-body](./docs/rule/no-aria-hidden-body.md)                                                 | ✅  |     | ⌨️  | 🔧  |
 | [no-attrs-in-components](./docs/rule/no-attrs-in-components.md)                                           | ✅  |     |     |     |
+| [no-autofocus-attribute](./docs/rule/no-autofocus-attribute.md)                                           | ✅  |     |     |     |
```

Comment:
this should include the emoji for accessibility.

Reply:
And the ✅ icon should go for now I believe, since it's not a recommended rule.